sdks/python/src/opik/cli/import_command.py

```diff
@@ -0,0 +1,592 @@
+
+            # Upload each data type
+            total_imported = 0
+
+            # Upload traces
+            if "traces" in data_types:
+                if debug:
+                    console.print("[blue]Uploading traces...[/blue]")
+                traces_imported = _import_traces(client, project_dir, dry_run, name)
+                total_imported += traces_imported
+
+            # Upload datasets
+            if "datasets" in data_types:
+                if debug:
+                    console.print("[blue]Uploading datasets...[/blue]")
+                datasets_imported = _import_datasets(client, project_dir, dry_run, name)
+                total_imported += datasets_imported
+
+            # Upload prompts
+            if "prompts" in data_types:
+                if debug:
+                    console.print("[blue]Uploading prompts...[/blue]")
+                prompts_imported = _import_prompts(client, project_dir, dry_run, name)
+                total_imported += prompts_imported
+
+            if dry_run:
+                console.print(
+                    f"[green]Dry run complete: Would import {total_imported} items[/green]"
+                )
+            else:
+                console.print(
+                    f"[green]Successfully imported {total_imported} items to project '{project_name}'[/green]"
+                )
+        else:
+            # Upload to all projects in workspace
+            # Get all projects in the workspace
+            try:
+                projects_response = client.rest_client.projects.find_projects()
+                projects = projects_response.content or []
+
+                if not projects:
+                    console.print(
+                        f"[yellow]No projects found in workspace '{workspace}'[/yellow]"
+                    )
+                    return
+
+                console.print(
+                    f"[blue]Found {len(projects)} projects in workspace[/blue]"
+                )
+
+                # Upload workspace-level data once (datasets, experiments, prompts)
+                total_imported = 0
+
+                # Upload datasets
+                if "datasets" in data_types:
+                    if debug:
+                        console.print("[blue]Uploading datasets...[/blue]")
+                    datasets_imported = _import_datasets(
+                        client, project_dir, dry_run, name
+                    )
+                    total_imported += datasets_imported
+
+                # Upload prompts
+                if "prompts" in data_types:
+                    if debug:
+                        console.print("[blue]Uploading prompts...[/blue]")
+                    prompts_imported = _import_prompts(
+                        client, project_dir, dry_run, name
+                    )
+                    total_imported += prompts_imported
+
+                # Upload traces to each project
+                if "traces" in data_types:
```

Comment:
Is there a real use case for it? I mean uploading the same traces to all projects?